atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -183,6 +183,7 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
                 if (locks.haveLocks()) {
                     sweptSuccessfully = runOnce();
                 } else {
+                    sweepMetrics.unableToAcquirePersistedLock();
```

Comment:
Oh yeah wrong ones. Looks like it should be checking in `PersistentLockManager#acquirePersistentLockWithRetry`. Will update.